### PR TITLE
Postgres enum PATCH

### PR DESCRIPTION
**Pull request by Vitexus**
https://github.com/robmorgan/phinx/issues/670

### CODE REVIEW

File: src/Phinx/Db/Adapter/PostgresAdapter.php [hunk 1]
```diff
@@ -44,6 +44,19 @@ class PostgresAdapter extends PdoAdapter implements AdapterInterface
      * @var array
      */
     protected $columnsWithComments = array();
+    /**
+     * Custom enums list
+     * 
+     * @var array 
+     */
+    private $enums = [];
```
Comment (robmorgan) [inline src/Phinx/Db/Adapter/PostgresAdapter.php]: we still support PHP 5.3, please use `array()`.

Comment (Vitexus) [inline src/Phinx/Db/Adapter/PostgresAdapter.php]: You right man!

**Comment by Vitexus** [comment]
Uff some work wait for me to make this pach make its job well ...

**Comment by shadowhand** [comment]
Thank you for contributing! The branch you made this PR against is no longer accepting patches. This probably happened because we didn't merge your changes fast enough. :sob:

Please create a new PR against the current default branch if you want to see these changes merged.

**Pull request closed without merging**